packages/firebase_messaging/android/src/main/java/io/flutter/plugins/firebasemessaging/FirebaseMessagingPlugin.java

```diff
@@ -153,8 +153,12 @@ public void onComplete(@NonNull Task<InstanceIdResult> task) {
                   channel.invokeMethod("onToken", task.getResult().getToken());
                 }
               });
+      result.success(null);
```

Comment:
This seems to be new behaviour, you probably added this to fix something else? Is it worth putting that change, along with a test driver test, in a separate PR?